zipkin.proto

```diff
@@ -212,3 +214,14 @@ message Annotation {
 message ListOfSpans {
   repeated Span spans = 1;
 }
+
+// Use this to report spans onto the gRPC transport. Implementations are asynchronous and may drop
+// spans for reasons of sampling or storage availability. While this is primarily used to store
+// spans, other operations may take place such as aggregation of service dependencies or data
+// cleaning.
+service SpanService {
+
+  // Unary call analogous to the POST /api/v2/spans endpoint. Spans are not required to be complete
```

Comment:
`Report the provided spans to the collector. Analogous to the...`

I don't think `Unary call` is needed since a non-unary call would have `stream` in the definition itself.

Reply:
aggreed